### PR TITLE
[WIP]Fix #2139 golint issues area/maya

### PR DESCRIPTION
**Pull request by abhishek-19**
This PR fixed the golint issue for the maya/cmd/maya-apiserver/app/command/start.go
fixes https://github.com/openebs/openebs/issues/2139
Added a comment for exported function NewCmdStart.

### CODE REVIEW

File: cmd/maya-apiserver/app/command/start.go [hunk 1]
```diff
@@ -94,6 +94,7 @@ type CmdStartOptions struct {
 	httpServer *server.HTTPServer
 }
 
+// NewCmdStart start mayaserver
```
Comment (princerachit) [inline cmd/maya-apiserver/app/command/start.go]: I think this is incorrect. This function returns a cobra command to start a server.

Comment (abhishek-19) [reply cmd/maya-apiserver/app/command/start.go]: Okay, I am checking for that.

**Comment by princerachit** [comment]
@abhishek-19 please provide complete link to issue in PR description

File: cmd/maya-apiserver/app/command/start.go (hunk 1 above)
Comment (sonasingh46) [reply cmd/maya-apiserver/app/command/start.go]: @abhishek-19 
Put it like this:
// NewCmdStart creates start command for maya-apiserver

**Comment by abhishek-19** [comment]
I have made that changes. Please review that. Thanks for the support @sonasingh46

File: types/v1/types.go [hunk 2]
```diff
@@ -162,6 +162,7 @@ const (
 type K8sKind string
 
 const (
+	// DeploymentKK is a K8s Deployment Kind
```
Comment (sonasingh46) [inline types/v1/types.go]: Can we please add period at the end of comment.

**Pull request closed without merging**